src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -83,10 +102,10 @@
       + "initial connection to discover the full cluster membership (which may change "
       + "dynamically), "
       + "this list need not contain the full set of servers (you may want more than one, though, "
-      + "in case a server is down).";
+      + "in case a server is down). While it is required to configure either of this or "
```

Comment:
oh, bootstrap servers can be found via zookeeper as well. hmm. but since we're trying to remove the zookeeper dependency, i don't understand why we want to support this? It appears you've added this support in this PR too, so it seems it's not for backwards compatibility (haven't looked into that thoroughly, could be mistaken).